### PR TITLE
ParaTimes: unit tests

### PR DESCRIPTION
**Pull request by buberdds**
Tests around
- form steps components/ unmocking `react-i18next` in a few tests to validate conditional interpolation
- paraTime saga 
- useParaTimesNavigation
- useParaTimes

PR adds:
- restriction around paratime route

### CODE REVIEW

File: src/app/pages/ParaTimesPage/TransactionAmount/__tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    expect(screen.getByRole('button', { name: 'Next' })).toBeDisabled()
+  })
+
+  it('should set input value to max', () => {
+    const setTransactionForm = jest.fn()
+    jest.mocked(useParaTimes).mockReturnValue({ ...mockUseParaTimesResult, setTransactionForm })
+    render(<TransactionAmount />)
+
+    userEvent.click(screen.getByRole('button', { name: 'MAX' }))
+    expect(setTransactionForm).toHaveBeenCalledWith(expect.objectContaining({ amount: '1' }))
```
Comment (lukaw3d) [inline src/app/pages/ParaTimesPage/TransactionAmount/__tests__/index.test.tsx:81]: Can you add precision tests with 1563114365108133939632n

File: src/app/useRouteHelper.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export const OpenWalletRequiredRoute = (props: TransitionRouteProps) => {
+  const history = useHistory()
+  const isAddressInWallet = useSelector(selectIsAddressInWallet)
+
+  useEffect(() => {
+    if (!isAddressInWallet) {
+      history.push(`/open-wallet`)
+    }
+  }, [isAddressInWallet, history])
```
Comment (lukaw3d) [inline src/app/useRouteHelper.tsx:15]: I expect something odd happens if user has two accounts in wallet, switch between them, and clicks back

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ParaTimesPage/TransactionAmount/__tests__/index.test.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/pages/ParaTimesPage/TransactionAmount/__tests__/index.test.tsx]: 2493300888889 isn't enough :D js numbers have that precision

Comment (buberdds) [reply src/app/pages/ParaTimesPage/TransactionAmount/__tests__/index.test.tsx]: updated in the second PR

**Pull request merged**